build/ps-entrypoint.sh

```diff
@@ -311,7 +319,7 @@ if [ "$1" = 'mysqld' -a -z "$wantHelp" ]; then
 			${monitorConnectGrant}
 
 			CREATE USER 'replication'@'%' IDENTIFIED BY '${REPLICATION_PASSWORD}';
-			GRANT SYSTEM_USER, REPLICATION SLAVE ON *.* to 'replication'@'%';
+			GRANT SYSTEM_USER, REPLICATION SLAVE, CONNECTION_ADMIN, BACKUP_ADMIN, GROUP_REPLICATION_STREAM ON *.* to 'replication'@'%';
```

Comment:
@Tusamarco can you check these new grants? are they OK with async replication too?

Reply:
How do u trigger the clone action? Because if u use the same user then CLONE_ADMIN is missed otherwise seems ok.